python/nano/src/bigdl/nano/tf/keras/inference/optimizer.py

```diff
@@ -314,8 +314,11 @@ def trace(model: Model,
         :param model: The Keras model to trace.
         :param accelerator: The accelerator to use, defaults to None meaning staying in Keras
                             backend. 'openvino' and 'onnxruntime' are supported for now.
-        :param input_spec: A (tuple or list of) tf.TensorSpec or numpy array defining the
-                           shape/dtype of the input.
+        :param input_spec: (optional) A (tuple or list of) ``tf.TensorSpec`` or numpy array
```

Comment:
we may remove numpy array here, and only support `tf.TensorSpec`

Reply:
> Or we should hide the usage of numpy array in the api doc and only suggest user to use tf.TensorSepc?

yes, we have no UT for numpy array, and there is no need to support two different methods.

Reply:
Have updated, please have a look again. Thank you!